etna/lib/etna/command.rb

```diff
@@ -39,6 +39,10 @@ def boolean_flags
       def string_flags
         @string_flags ||= []
       end
+
+      def multi_flags
+        @multi_flags ||= []
```

Comment:
I added the ability to include flags that are repeatable in etna.

For instance

`etna my bommand --env A=1 --env B=2` would result in env: ["A=1", "B=2"]

I was planning on using this for a command line demo of the backend, but didn't end up completing it.